src/components/atoms/Alert/Alert.tsx

```diff
@@ -24,29 +25,33 @@ export interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
 const Wrapper = styled.div<{ severity: Severity; inline: boolean; hasRoundedCorners: boolean }>`
   display: ${({ inline }) => (inline ? 'inline-flex' : 'flex')};
   position: relative;
-  padding: 8px 12px 8px 12px;
+  align-items: ${({ severity, theme }) => theme.alert[severity].alignItems};
+  padding: ${({ severity, theme }) => theme.alert[severity].padding ?? '8px 12px 8px 12px'};
   background: ${({ severity, theme }) => theme.alert[severity].background};
   color: ${({ severity, theme }) => theme.alert[severity].text};
-  font-size: ${typography.sizes.text.body};
-  line-height: ${typography.sizes.lineHeight.body};
-  font-family: ${typography.primary};
-  font-weight: 400;
-  border-radius: ${({ hasRoundedCorners }) => (hasRoundedCorners ? 4 : 0)}px;
+  font-size: ${({ theme }) => theme.typography.text.sizes.body ?? typography.sizes.text.body};
+  line-height: ${({ theme }) => (theme.typography.lineHeight.small ? typography.sizes.lineHeight.body : '24px')};
```

Comment:
this one should just use the value from theme e.g. `line-height: ${({ theme }) => (theme.typography.lineHeight.small ?? typography.sizes.lineHeight.body`

Reply:
Nice Thank you.